contracts/multisig-prover/src/encoding/abi2/mod.rs

```diff
@@ -143,14 +143,34 @@ fn evm_address(pub_key: &MultisigPublicKey) -> Result<Address, ContractError> {
     }
 }
 
+pub fn make_operators(worker_set: WorkerSet) -> Operators {
```

Comment:
Took from [abi::make_operators](https://github.com/axelarnetwork/axelar-amplifier/blob/multisig-prover-v0.2.0/contracts/multisig-prover/src/encoding/abi.rs#L133) and add a unit test